ci/spack/cpu.yaml

```diff
@@ -3,7 +3,7 @@ spack:
     - sirius@develop %gcc +vdwxc +tests +apps build_type=RelWithDebInfo +elpa ^openblas %gcc ^mpich
     - sirius@develop %gcc +vdwxc +tests +apps build_type=RelWithDebInfo +elpa ^openblas %gcc ^openmpi
     - sirius@develop %clang +tests +apps build_type=RelWithDebInfo ^openblas ^mpich ~fortran
-  view: false
```

Comment:
I don't think this will work, cause there's multiple flavors of sirius being built (%gcc, %clang, ^openmpi, ^mpich), so they will clash with filenames. You can do something like

```
view:
  gcc-with-mpich:
    root:/root/local/gcc-with-mpich
    select: ['%gcc ^mpich']
  gcc-with-openmpi:
    root:/root/local/gcc-with-openmpi
    select: ['%gcc ^openmpi']
  clang-with-mpich:
    root:/root/local/clang-with-mpich
    select: ['%clang ^openmpi']
```

to avoid the clashes, but that seems a bit verbose